sw/device/rom_exts/manifest.md

```diff
@@ -184,13 +184,35 @@ Notes:
     This is used when signing and validating the image. This happens in the
     Mask ROM, as well as during firmware update.
 
-1.  **Usage Constraints** This is a 32-bit enumeration value which denotes what
-    kinds of devices the ROM_EXT image may be used on.
-
-    This is used by the Mask ROM to validate the image is running on the right
-    kind of hardware.
-
-    **Open Q** What possible values should this have?
+1.  **Usage Constraints** This denotes which pieces of the OpenTitan Device
```

Comment:
The device identifier does not contain the SKU ID fuses. These should eventually be added her as well once we agree on a format.

Reply:
So we need to support more inputs than just the 256b field described "Device Identifier" in the "Identities and Root Keys" document? Have the additional inputs been described anywhere for us to look at? Or do we know their width?

Reply:
> The device identifier does not contain the SKU ID fuses. These should eventually be added her as well once we agree on a format.

In Identities and Root Keys the second 128b of the Device ID is described as:

```
128b SKU specific device information
The device provisioner information varies for each provisioning use case.
```

Is this not the SKU ID? Or is that expected to live alongside this in OTP but _not_ inside the Device ID?

Reply:
That one was allocated for per use case specific global identifier requirements.  The labeling was confusing, need to adjust that.

Regarding where the data is located in OTP. We are a bit early to answer that question. We will eventually have to document the OTP memory allocations and define what SW config options we want to encode in the SW config partition.

Also, this intersects the `otp_settings` use case in the ownership transfer doc. It would be good to discuss if we are planning to use a similar approach in mask rom. In addition to device_id, and sku information, there are things such as fuse page locks and end of manufacturing bits that we may need to check for.

Reply:
I *think* this is now addressed by being less specific about where the 8192 bits come from.